assets/scss/_variables_project.scss

```diff
@@ -1,12 +1,19 @@
 /* Updated color scheme to match the background image */
 
 $in-toto-colors: (
-  'orange': #ed4b27,       // Keep existing
-  'orange-light': #f47a39, // Keep existing
-  'blue': #1b2838,         // Darker blue for better contrast
-  'blue-light': #a3b5c8,   // Lighter blue for accentss
+  'orange': #ed4b27,
+  // Keep existing
+  'orange-light': #f47a39,
+  // Keep existing
+  'blue': #1b2838,
+  // Darker blue for better contrast
+  'blue-light': #a3b5c8,
+  // Lighter blue for accentss
```

Comment:
I _think_ that if you use `/* ... */` comments, Prettier will leave them at the end of the line. If not, then leave this as is.